docs/ref/csrf.txt

```diff
@@ -136,18 +136,30 @@ and read the token from the DOM with JavaScript:
 .. code-block:: html+django
 
     {% csrf_token %}
-    <script type="text/javascript">
-    // using jQuery
-    var csrftoken = jQuery("[name=csrfmiddlewaretoken]").val();
+    <script>
+    var csrftoken = document.querySelector('[name=csrfmiddlewaretoken]').value;
     </script>
 
 Setting the token on the AJAX request
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Finally, you'll have to actually set the header on your AJAX request, while
-protecting the CSRF token from being sent to other domains using
-`settings.crossDomain <https://api.jquery.com/jQuery.ajax/>`_ in jQuery 1.5.1
-and newer:
+Finally, you'll have to actually set the header on your AJAX request. Using the
+`fetch()
+<https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/fetch>`_
+API:
+
+.. code-block:: javascript
+
+    var request = new Request(/* url */);
+    var headers = new Headers();
+    headers.set('X-CSRFToken', csrftoken);
+    fetch(url, {method: 'POST', headers: headers}).then(function(response) {
```

Comment:
Should this say `fetch(request, ...`?

Reply:
Yup, you're right. Fixed in latest revision.